orca-front50/src/test/groovy/com/netflix/spinnaker/orca/front50/config/Front50ConfigurationSpec.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2024 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.front50.config
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.okhttp.OkHttpClientConfigurationProperties
+import retrofit.Endpoint
+import spock.lang.Specification
+import spock.lang.Subject
+import java.util.concurrent.TimeUnit
+
+/**
+ * Tests for Front50 timeout configuration
+ */
+class Front50ConfigurationSpec extends Specification {
```

Comment:
Would you mind converting this to java please.  The less groovy code we have in the world, the better.

Reply:
Yes, let me know if it looks better now :)

Reply:
Still looks like groovy to me...